voicevox_engine/metas/MetasStore.py

```diff
@@ -0,0 +1,50 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+from voicevox_engine.metas.Metas import SpeakerSupportPermittedSynthesisMorphing
+
+
+class MetasStore:
```

Comment:
ちょっと説明足しておくと将来さまようコミッターを減らせるかもと思いました！
こんな感じでしょうか。
```suggestion
class MetasStore:
    """
    話者やスタイルのメタ情報を管理する
    """
```